src/quo2/components/buttons/slide_button/utils.cljs

```diff
@@ -0,0 +1,38 @@
+(ns quo2.components.buttons.slide-button.utils
+  (:require
+    [quo2.components.buttons.slide-button.constants :as constants]
+    [quo2.foundations.colors :as colors]))
+
+;; Utils
```

Comment:
nit: this comment is not so necessary imo as these are utils as described by the namespace 👍

Reply:
done